app/Http/Controllers/TransfersController.php

```diff
@@ -316,9 +316,11 @@ public function audioImportUpload(Request $request)
  
       $import = new AudioImport($filePath);
       $data = $import->data();
+      $possibleDataKeys = Transfer::AUDIO_IMPORT_KEYS;
+      $tableId = 'audio-upload-data-container';
```

Comment:
I think the $tableId vars all follow the same format in each of the controllers, so if you wanted, you could pass in just "audio" or "items" to the partial and render "{{dataType}}-upload-data-container" in the view, but that may be overkill/maybe you have exceptions to this format.

Reply:
Same with $divId, I think

Reply:
👍